rows.go

```diff
@@ -127,8 +127,13 @@ func checkTypeValue(columnType string, val interface{}) error {
 }
 
 func extractStructColumn(columnType string) (string, string, error) {
+	columnType = strings.TrimSpace(columnType)
+	if idx := strings.IndexRune(columnType[1:], '`'); strings.HasPrefix(columnType, "`") && idx != -1 {
+		// We use idx+2 since we found this index in the substring starting from the second character
+		return strings.Trim(columnType[1:idx+2], " `"), strings.TrimSpace(columnType[idx+2:]), nil
+	}
 	field := strings.SplitN(strings.TrimSpace(columnType), " ", 2)
-	if len(field) != 2 {
+	if len(field) < 2 {
```

Comment:
also, would there be a case when we have > 2 here?

Reply:
We do strings.SplitN(..., 2) which says at most 2. The only way this can fail is we can have 1 item here (which will mean there are no whitespaces in a string -> it's malformed). This confused me for a second when can it be !=2 here, so I made a more specific check